CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Flutter Plugin Changelog
 
+## Version 8.0.0 - October 24, 2024
+
+Major version release with new Android plugin extender method: `onAirshipReady`.
```

Comment:
Major version that makes it easier to include Airship in a hybrid app. The only breaking change is when extending the `AirshipPluginExtender` protocol on java there is a new `extendConfig(Contex, AirshipConfigOptions.Builder)` method to implement. Most application will not be affected. 

### Changes
- Added new methods to the plugin extender to make hybrid app integrations easier